objectscale-portal/values.yaml

```
@@ -107,4 +107,4 @@ hooks:
   image:
     repository: ecs-flex-graphql
     tag: 0.82.0-769.a415d196 # rfw-update-this graphql-imagee
```

```suggestion
    tag: 0.82.0-769.a415d196 # rfw-update-this graphql-image
```
fix the rfw comment otherwise it won't be auto-updated during release.